ftw/news/profiles/show_on_homepage/metadata.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0"?>
+<metadata>
+    <version>1000</version>
+    <dependencies>
+        <dependency>profile-ftw.news:default</dependency>
```

Comment:
I would prefer that `show_on_homepage` does not include (depend on) `default` for a practical reason:
When I already have an `ftw.news` installation running, which I may have customized (e.g. changed the view name), installing the `show_on_homepage` feature will reset all ftw.news components to the default settings. I then have to know that I have to reconfigure it.

Not having the dependency on the other hand would make the profile not work at all when solely installed, so the integrator will figure out that he needs to install the `default` profile too.

It would be great though when the readme contains a hint about it.